cmd/ocm/cluster/login/login.go

```diff
@@ -47,13 +49,8 @@ var Cmd = &cobra.Command{
 
 func init() {
 	flags := Cmd.Flags()
-	flags.StringVarP(
-		&args.user,
-		"username",
-		"u",
-		"",
-		"Username, will prompt if not provided",
-	)
+	flags.StringVarP(&args.user, "username", "u", "", "Username, will prompt if not provided")
+	flags.BoolVarP(&args.console, "console", "", false, "Open the OpenShift console for the cluster")
```

Comment:
I think this help text should explicitly indicate that this will open the console in a browser.